rskj-core/src/main/java/org/ethereum/solidity/SolidityType.java

```diff
@@ -33,6 +33,14 @@
 import java.util.List;
 
 public abstract class SolidityType {
+
+    public static final String BYTES = "bytes";
+    public static final String STRING = "string";
+    public static final String ADDRESS = "address";
+    public static final String BOOL = "bool";
+    public static final String INT = "int";
+    public static final String UINT = "uint";
```

Comment:
Why not to use `uint256`? `uint`is only a convenience alias in Solidity. The text to use in the hash calculation of event signature (and methods) is `uint256`. Same with `int`

Reply:
You are probably right, but these changes are only part of making the code more legible. If I change that probably I will have to change many tests, and it becomes out of scope.
You can create an issue with that with "good first issue" tag.

Reply:
Are you sure that the event signature are generated using uint256 instead of uint? An event has a topic, that is the hash of the signature, and Solidity, for event declared as EventX(uint), uses EventX(uint256) as input string for event signature hash

Reply:
I just reviewed the code, and thanks to SolidityType is abstract, the supplied `uint` name is never used as canonical name.

But, could we add a test that assert that the hash signateure generated by these events IS THE same hash signature that would be generated by a Solidity program?